packages/components/src/public/QueryModel/GridPanel.tsx

```diff
@@ -1192,6 +1192,6 @@ interface GridPanelWithModelProps<T = {}> extends GridPanelProps<T> {
  */
 export const GridPanelWithModel: FC<GridPanelWithModelProps> = memo(({ queryConfig, ...props }) => {
     const id = useMemo(() => queryConfig.id ?? createQueryModelId(queryConfig.schemaQuery), [queryConfig]);
-    const queryConfigs = useMemo(() => ({ [id]: queryConfig }), [id, queryConfig]);
+    const queryConfigs = useMemo(() => ({ [id]: { ...queryConfig, includeTotalCount: true } }), [id, queryConfig]);
```

Comment:
nit: This seems like something that should be done within the usages, who are providing the `queryConfig` rather than have this opinion in `GridPanelWithModel`.

Reply:
I went back and forth on this. There are approx 10 usages of GridPanelWithModel and putting this prop here "fixed" things for all those cases without having to touch each. Do you think there is actually a case where the usage of GridPanelWithModel would not want to load this? Seems that like is always needed/wanted in the grid case.

Reply:
OK, nvm. I found a case. The LKB RelatedData.tsx component uses the `GridPanelWithModel` component to load a set of rows with a specific IN clause filter and sets the `showButtonBar={false}` so it doesn't need the total row count value. Reverting this and applying prop to usages.